research/format.markdown

```diff
@@ -0,0 +1,138 @@
+## Draft 2 to Draft 3:
+
+> Behaviour: Not required to validate → MAY validate(but aren’t required to)
```

Comment:
Please avoid when possible non ASCII characters like `→`. You can just say before/after or something like that